galaxy_ng/app/access_control/statements/standalone.py

```diff
@@ -94,9 +94,15 @@
         },
         {
             "action": ["update", "partial_update"],
-            "principal": "authenticated",
+            "principal": ["authenticated"],
+            "effect": "allow",
+            "condition": "is_current_user"
```

Comment:
We don't want to allow users to update themselves on the `UserViewSet`. The `UserViewSet` contains the ability to update a user's groups, which means that if user's can update themselves from this API endpoint, they can change their groups and elevate their permissions. That's why we have a separate access policy for `MyUserViewSet`